class.jetpack-module.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Class Jetpack_Module
+ *
+ */
+class Jetpack_Module {
+	public $slug;
+
+	public $name;                       // Module Name
+	public $description;                // Module Description
+	public $jumpstart_desc;             // Jumpstart Description
+	public $sort;                       // Sort Order
+	public $recommendation_order;       // Recommendation Order
+	public $introduced;                 // First Introduced
+	public $changed;                    // Major Changes In
+	public $deactivate;                 // Deactivate
+	public $free;                       // Free
+	public $requires_connection;        // Requires Connection
+	public $auto_activate;              // Auto Activate
+	public $module_tags;                // Module Tags
+	public $feature;                    // Feature
+	public $additional_search_queries;  // Additional Search Queries
+	public $plan_classes;               // Plans
+
+	public function __construct( $module, $args = array() ) {
+		$this->slug = $module;
+
+		$this->set_props( $args );
+	}
+
+	private function set_props( $args ) {
+		$default_args = array(
```

Comment:
I would set the default when we declare each object property.
i.e:
`public $jumpstart_desc = '';`

Reply:
I am not sure you can set some defaults as arrays in lower PHP version.